### DISCUSSION TITLE
onlyoffice的显示出问题是为啥？

### DISCUSSION THREAD

**Discussion by monobo1989** [AlistGo/alist#6722] [Q&A]
![QQ图片20240709075523](https://github.com/alist-org/alist/assets/73515609/37ad36e5-a58b-419e-8088-a40cd49ae8e6)
这个提示问题出在哪呢？

![QQ图片20240709075652](https://github.com/alist-org/alist/assets/73515609/0ea83f95-4b20-4f33-8255-57ec5cc775c2)

**Comment by pongfcnkl** [comment]
如果能下载那就是你onlyoffice的问题
